Client/Application/LeanplumIntegration.swift

```diff
@@ -51,7 +51,7 @@ enum UserAttributeKeyName: String {
 
 private enum SupportedLocales: String {
     case US = "en_US"
-    case DE = "de"
```

Comment:
So should we have both?

Reply:
Not sure. I'll double check what happens if you have english as your langauge and german as your region.

Reply:
if your language is set to english and region is set to Germany your identifier is en_DE.